packages/react-server/core/renderMiddleware.js

```diff
@@ -185,7 +185,7 @@ function renderPage(req, res, context, start, page) {
 
 	var routeName = context.navigator.getCurrentRoute().name;
 
-	logger.debug("Route Name: " + routeName);
+	logger.debug("Route Name: "  routeName);
```

Comment:
:squint:  Something fishy's going on here.  Did this PR just remove all of the `+` characters?